### PR TITLE
adjust sending user messages

### PR DESCRIPTION
**Pull request by vrabeshko**

### CODE REVIEW

**Comment by vrabeshko** [comment]
[S - adjust SKD's to new conversations API](https://trello.com/c/Unny1Dwc/1255-s-adjust-skds-to-new-conversations-api)

File: src/xcoobee/sdk/Users.js [hunk 1]
```diff
@@ -203,8 +203,12 @@ class Users {
    *
    * @async
    * @param {string} message
-   * @param {*} consentId
-   * @param {*} breachId
+   * @param {Reference} reference - at least one of following keys should be provided
+   * @property {string} consentId
+   * @property {string} breachId
```
Comment (dhurlburtusa) [inline src/xcoobee/sdk/Users.js]: In `ConversationsApi.sendUserMessage`, I only see `consentId`, `requestRef`, and `ticketId` being used. Should `breachId` and `complaintId` be removed from this doc comment?

**Comment by dhurlburtusa** [comment]
@vrabeshko I don't have access to the Trello card attached so I can't update it.

**Comment by vrabeshko** [comment]
@dhurlburtusa I applied requested changes. Also in this PR I commited fixes for integration tests. Check pls

**Comment by vrabeshko** [comment]
[M - Fix integration tests on JS SDK](https://trello.com/c/KheRJPC9/1240-m-fix-integration-tests-on-js-sdk)

**Pull request merged**